src/test/java/org/dcsa/api_validator/tnt/v2/GetEventsTest.java

```diff
@@ -99,26 +100,31 @@ public void testShipmentEventsQueryParam() {
     @Test
     public void testEquipmentReferenceQueryParam() {
 
-        List<String> equipmentReferences = getListOfAnyAttribute("equipmentReference","eventType","EQUIPMENT");
-
-        for (String equipmentReference : equipmentReferences) {
+        List<String> EquipmentReferences = getListOfAnyAttribute("equipmentReference","eventType","EQUIPMENT");
 
+        for (String EquipmentReference : EquipmentReferences) {
+            System.out.println("eqr");
+            System.out.println(EquipmentReference);
```

Comment:
Please remove system out...

Reply:
Will do, I will fix the variable naming too.